<robot name="rallycar">
  <link name="base_link"/>
  <link name="laser"/>
  <link name="imu_frame"/>
  <link name="rs_t265_pose_frame"/>

  <joint name="base_to_laser" type="fixed">
    <parent link="base_link"/>
    <child link="laser" />
    <!--The same frame-->
    <origin xyz="0.285 0.0 0.15" rpy="0.0 0.0 0.0"/>
  </joint>

  <joint name="base_to_imu" type="fixed">
    <parent link="base_link"/>
    <child link="imu_frame" />
    <origin xyz="0.045 -0.045 0.13" rpy="0.0 0.0 0.0"/>
  </joint>

  <joint name="base_to_real_sense" type="fixed">
    <parent link="base_link"/>
    <child link="rs_t265_pose_frame" />
    <origin xyz="0.325 0.0 0.13" rpy="0.0 0.0 0.0"/>
  </joint>
</robot>
